Mail/Views/Switch User/AddMailboxView.swift

```diff
@@ -0,0 +1,146 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCoreUI
+import InfomaniakDI
+import MailCore
+import MailResources
+import SwiftUI
+
+struct AddMailboxView: View {
+    enum FocusedField {
+        case mailAddress, hiddenPassword, visiblePassword
+    }
+
+    @Environment(\.dismiss) var dismiss
+
+    var completion: (Mailbox?) -> Void
+
+    @State private var newAddress = ""
+    @State private var password = ""
+    @State private var isSecured = true
+    @FocusState private var focusedField: FocusedField?
+    @State private var showError = false
+
+    private var buttonDisabled: Bool {
+        return !Constants.isEmailAddress(newAddress) || password.isEmpty
+    }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(MailResourcesStrings.Localizable.attachAddressDescription)
+                .textStyle(.bodySecondary)
+                .padding(.bottom, 32)
+
+            TextField(MailResourcesStrings.Localizable.attachAddressInputHint, text: $newAddress)
```

Comment:
![Simulator Screenshot - iPhone 14 Pro - 2023-05-02 at 09 18 41](https://user-images.githubusercontent.com/25110022/235603795-debd1027-c1ef-4189-b16b-d28bdc4cdb3b.png)
I have a regular keyboard and not the one adapted for mail addresses